network-api/networkapi/wagtailpages/models.py

```diff
@@ -600,23 +607,24 @@ class BlogPage(FoundationMetadataPageMixin, Page):
         ('quote', customblocks.QuoteBlock()),
     ])
 
-    # Editor panels configuration
+    tags = ClusterTaggableManager(through=BlogPageTag, blank=True)
+
+    zen_nav = True
 
     content_panels = Page.content_panels + [
         FieldPanel('author'),
         StreamFieldPanel('body'),
     ]
 
-    # Promote panels configuration
-    tags = ClusterTaggableManager(through=BlogPageTag, blank=True)
-
     promote_panels = FoundationMetadataPageMixin.promote_panels + [
         FieldPanel('tags'),
     ]
 
-    # Database fields
-
-    zen_nav = True
+    api_fields = [
```

Comment:
these will show up in the API request **if** `?types=...` is specified with `wagtailpages.BlogPage` as one of the types, **and** `?fields=*` is used (or `?fields=title,author,tags`, but that would break selecting children of different types)

Reply:
For more details, see http://docs.wagtail.io/en/v2.1.1/advanced_topics/api/v2/usage.html